tests/inspect/inspect.go

```diff
@@ -126,6 +127,7 @@ func init() {
 	globalFlagset.BoolVar(&globalFlags.CheckMountNS, "check-mountns", false, "Check if app's mount ns is different than stage1's. Requires CAP_SYS_PTRACE")
 	globalFlagset.BoolVar(&globalFlags.PrintNoNewPrivs, "print-no-new-privs", false, "print the prctl PR_GET_NO_NEW_PRIVS value")
 	globalFlagset.StringVar(&globalFlags.CheckMknod, "check-mknod", "", "check whether mknod on restricted devices is allowed")
+	globalFlagset.BoolVar(&globalFlags.PrintOOMScoreAdj, "print-oom-score-adj", false, "print /proc/self/oom_score_adj")
```

Comment:
This should also doable just via `--read-file` and `--file-name /proc/self/oom_score_adj`. Is there a reason for having this specifically?